src/Components/WebView/Platforms/WindowsForms/testassets/WinFormsTestApp/Form1.cs

```diff
@@ -0,0 +1,25 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System.Windows.Forms;
+using System.Net.Http;
+using Microsoft.AspNetCore.Components.WebView.WindowsForms;
+using Microsoft.Extensions.DependencyInjection;
+
+namespace WinFormsTestApp
+{
+    public partial class Form1 : Form
```

Comment:
Give it a proper name, otherwise it's like `Class1` and people will ignore it or delete it.

Reply:
I was following on from the conventions in other projects like in `BlazorWinFormsApp`. Doesn't seem problematic right now but we could certainly change them all at some point.